{"name": "Проверка ошибки при авторизации с неправильным логином или паролем", "status": "passed", "description": "Проверяется ошибка при вводе неправильного логина или пароля.", "parameters": [{"name": "login_password", "value": "{'login': 'wronglogin', 'password': 'dab0VnJG$%'}"}], "start": 1723375731976, "stop": 1723375732121, "uuid": "82121870-1138-4351-bc4a-3b7e4b265680", "historyId": "a367640ecf5d8c9fcca8dc5f0d55fc35", "testCaseId": "432e0cc92bda3737e4e249e8e3172c47", "fullName": "tests.test_courier_login.TestCourierLogin#test_courier_login_invalid_credentials", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_courier_login"}, {"name": "subSuite", "value": "TestCourierLogin"}, {"name": "host", "value": "HOME-PC"}, {"name": "thread", "value": "6908-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_courier_login"}]}